File: JAVA/JavaOdev2/RASTGELEKISIURET/src/Kisiler/Telefon.java
/** 
 * 
 * @author Burak Yıldırım
 *         M.İbrahim Turan
 * @since 01.04.2018
 * <p>
 * TC kimlik ve IMEI Kontrolü yapan program simulasyonu
 * </p>
 */

package Kisiler;

import java.util.Random;

public class Telefon 
{
    private int telNO[] = new int[11];
    
    public IMEINo imei = new IMEINo() ;
   
   public int[] telNoGetir()
   {
       return telNO;
   }
   
   public void telNoAta(int[] index)
   {
       System.arraycopy(index, 0 , telNO ,0 , 11);
   }
   
   public void telNoUret()
    {
        Random rand = new Random();
        telNO[0] = 0 ;
        telNO[1] = 5 ;
        while(true)
        {
           if(6 > telNO[2] && telNO[2] > 2  )
                    break;
           else
               telNO[2] = rand.nextInt(10);
        }
        telNO[3] = rand.nextInt(10);
        for(int i = 4 ; i < telNO.length ; i++ )
        {
            telNO[i] = rand.nextInt(10);
        }
    }
  
    
}

File: C/Odev3-PDP/src/Telefon.c
/** 
 * 
 * @author Burak Yıldırım
 *         M.İbrahim Turan
 * @since 22.04.2018
 * <p>
 * TC kimlik ve IMEI Kontrolü yapan program simulasyonu
 * </p>
 */

#include "Telefon.h"
#include "time.h"
#include "stdio.h"
#include "stdlib.h"

void telNoUret(telefon t)
{
	
	
	
	
	t->telNO[0] = 0 ;
    t->telNO[1] = 5 ;
        
	while(true)
    {
        if(6 > t->telNO[2] && t->telNO[2] > 2  )
            break;
        else
            t->telNO[2] = rand() % 10 ;
    }
        
	t->telNO[3] = rand() % 10 ;
        
	for(int i = 4 ; i < 11 ; i++ )
    {
        
		t->telNO[i] = rand() % 10 ;
		
    }
	
}

void telNoAta(const telefon t , int index[11])
{
	for(int i = 0 ; i < 11 ; i++)
	{
		
		t->telNO[i] = index[i];
		
	}
	
	
	
}

telefon telefonUret()
{
	telefon t ;
	t=(telefon)malloc(sizeof(struct TELEFON));
	telNoUret(t);
	t->i = imeiNoUret();
	
}